MyCryptoHQ/MyCrypto
# Ak 2124 disabled verify for invalid address

Closes #2124

### Description

When verifying a message, you should not be able to select the button "Verify Message" until you have inputted a valid message to verify.

### Changes

* On paste checks if the signautre is valid to enable or disable the verify message button
* On change (user typing) also checks to see if the signature is valid
* You Made

### Steps to Test

1. Paste or type in an invalid signature and the "verify message" button should be disabled still.
2. Paste or type in a valid signature and the "verify message" button should be enabled. 
3. If you delete or change a valid signature the "verify message" button should go back to disabled. 

### Screenshots
Before: https://user-images.githubusercontent.com/25834495/44283848-a4634f80-a22d-11e8-933e-ad00384effbc.png

After:  (Invalid signature) https://i.imgur.com/JnehNsm.png
